OpenAI/Packages/com.openai.unity/Documentation~/README.md

```diff
@@ -226,6 +229,34 @@ await foreach (var token in api.CompletionsEndpoint.StreamCompletionEnumerableAs
 }
 ```
 
+### [Chat](https://platform.openai.com/docs/api-reference/chat)
+
+Given a chat conversation, the model will return a chat completion response.
+
+#### [Chat Completions](https://platform.openai.com/docs/api-reference/chat/create)
+
+Creates a completion for the chat message
+
+```csharp
+IClient(OpenAIAuthentication.LoadFromEnv());
+var chatPrompts = new List<ChatPrompt>
+{
+new ChatPrompt("system", "You are a helpful assistant."),
+new ChatPrompt("user", "Who won the world series in 2020?"),
+new ChatPrompt("assistant", "The Los Angeles Dodgers won the World Series in 2020."),
+new ChatPrompt("user", "Where was it played?"),
+};
+var chatRequest = new ChatRequest(chatPrompts);
+var result = await api.ChatEndpoint.GetCompletionAsync(chatRequest);
+Debug.Log(result.Fi
```

Comment:
unfinished line?

Reply:
Yeah I already knocked this out in main. Thanks